packages/function-once/index.js

```diff
@@ -0,0 +1,20 @@
+module.exports = once;
+
+/*
+let i = 0;
+const getFirst = once(() => ++i);
+getFirst(); // 1
+getFirst(); // 1
+*/
+
+function once(fn) {
```

Comment:
One more thing: can you test that `fn` is a function and throw `new Error("expected a function but got " + fn)`. Then add a couple more tests where you pass undefined or an object and make sure it throws.

e.g. something like
` t.throws(function () {once(undefined)});`

Thanks! and sorry for the delay getting to this.